### DISCUSSION TITLE
Changing civilian faction

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1537] [Question and answer] [answered]
I tried changing by editing the param, but it didn't work. I want to ask if there's any other way or if i messed anything up. Thanks.

**Comment by Vdauphin** [comment]
Hello,
It is the correct way
Cheers

**Reply by XDSenDX** [reply]
Hello i tried it but still not working. I also tried the other method by manually putting in class name but they aren't spawning and there is an error with creating suicide bomber.
Here's what i did:
//Save class name to global variable
btc_civ_type_units = [LOP_Tak_Civ_Man_01], [LOP_Tak_Civ_Man_06], [LOP_Tak_Civ_Man_08], [LOP_Tak_Civ_Man_07], [LOP_Tak_Civ_Man_05], [LOP_Tak_Civ_Man_02], [LOP_Tak_Civ_Man_10], [LOP_Tak_Civ_Man_09], [LOP_Tak_Civ_Man_11], [LOP_Tak_Civ_Man_12], [LOP_Tak_Civ_Man_04], [LOP_Tak_Civ_Man_14], [LOP_Tak_Civ_Man_13], [LOP_Tak_Civ_Man_15], [LOP_Tak_Civ_16];
//allclasse = [[_p_civ_veh]] call btc_civ_fnc_class;
btc_civ_type_veh = [LOP_Tak_Civ_Hatchback], [LOP_Tak_Civ_Landrover], [LOP_Tak_Civ_Offroad], [LOP_Tak_Civ_UAZ], [LOP_Tak_Civ_UAZ_Open], [LOP_Tak_Civ_Ural], [LOP_Tak_Civ_Ural_Open];
btc_civ_type_boats =[LOP_Tak_Civ_Hatchback] ;

**Reply by Vdauphin** [reply]
Classname are not global variable, there are string so you need to write them like this `"LOP_Tak_Civ_Man_01"`
Also `btc_civ_type_units` need a table of element, what you wrote doesn't make sense because it is a sequence of array

It looks like you are lacking of basic coding knowledge, I would highly recommend you to use the intended way or join the Arma official Discord for help

**Comment by XDSenDX** [comment]
> Classname are not global variable, there are string so you need to write them like this `"LOP_Tak_Civ_Man_01"` Also `btc_civ_type_units` need a table of element, what you wrote doesn't make sense because it is a sequence of array
> 
> It looks like you are lacking of basic coding knowledge, I would highly recommend you to use the intended way or join the Arma official Discord for help

Thanks, i'll consider.

**Reply by XDSenDX** [reply, marked as answer]
Update: I got it working somehow just by following the other method.
Cheers.
